pyramid_openapi3/tests/test_views.py

```diff
@@ -181,3 +182,166 @@ def test_openapi_view_validate_HTTPExceptions() -> None:
 
         with pytest.raises(HTTPForbidden):
             view(context, request)
```

Comment:
Hey @gweis I need your eyes on this: there used to be a line like this: `assert str(exc.value) == "Unknown response http status: 403"`, but is was removed in later commits in #26.

Looking at the `test_openapi_view_validate_HTTPExceptions` test case, it seems that it is not testing anything? At least the comments say it's testing that un-spec-ed responses should be re-raised as `InvalidResponse` but there are not asserts to confirm this?

Reply:
Good catch. (totally overlooked that doc string).

The difference is, that the response is now validated inside the tween (not in the view wrapper).
So either this type of test should be done inside `test_validation.py`, where tests are run through a pyramid `Router` or this tests needs to be update. One way to do that would be like this: https://github.com/gweis/pyramid_openapi3/commit/7dc65ca45ebbcbaaa9907c1b9bf11d0a844c5d2e

There is one small issue with the tween. It only validates the response correctly if there is an exception view handler configured in the pyramid app. Usually this is the case per default, but the normal pyramid testing test setup does not instal one. (I'll create another pull request to handle that case as well)